### ISSUE TITLE
version from github < CRAN version

### ISSUE THREAD

**Issue by xhudik** [ebailey78/shinyBS#77]
Hi there,
once you install shinyBS from cran you get version 0.61; however if you get it from github

```
devtools::install_github("ebailey78/shinyBS")
```

the version is 0.25
Not sure where this can be changed ...

**Comment by xhudik** [comment]
ok, problem is here:

```
devtools::install_github("ebailey78/shinyBS")
Downloading GitHub repo ebailey78/shinyBS@master
from URL https://api.github.com/repos/ebailey78/shinyBS/zipball/master
Installing shinyBS
...
```

if you download: https://api.github.com/repos/ebailey78/shinyBS/zipball/master - you will get the older version (different than `git clone https://github.com/ebailey78/shinyBS` ). Can we synchronized these 2 sources?

**Comment by tiagochst** [comment]
The version in master is 0.25, you can install version 0.62 from another branch
`devtools::install_github("ebailey78/shinyBS", ref="shinyBS3")`

**Comment by xhudik** [comment]
yes, this is what I found out (after a long time :)) - see my previous comment , but I dont think there is many people around who would even realize they are using an old version. Therefore I've suggested to merge  shinyBS3 branch into master...
